packages/api-server/src/methods/modules/eth.ts

```diff
@@ -786,15 +792,25 @@ export class Eth {
       // Convert polyjuice tx to api transaction
       const { tx, fromAddress }: AutoCreateAccountCacheValue =
         JSON.parse(polyjuiceRawTx);
-      const apiTransaction: EthTransaction =
-        polyjuiceRawTransactionToApiTransaction(
-          tx,
-          ethTxHash,
-          tipBlock.hash,
-          tipBlock.number,
-          fromAddress
-        );
-      return apiTransaction;
+      const isTxExists: boolean | Hash = await this.isAcaTxExist(
```

Comment:
```suggestion
      const isAcaTxExist: boolean = await this.isAcaTxExist(
```

Reply:
fun `isAcaTxExist` won't return a Hash type.

Reply:
> fun `isAcaTxExist` won't return a Hash type.

Yes, this interface updated and forget to update type here.